common/flavorization/wc_flavorization.txt

```diff
@@ -0,0 +1,39 @@
+﻿#Princes (children of Kings/Emperors, sorted by culture)
+
+### Dragons ###
+prince_male_dragon_kingdom {
+	type = character
+	gender = male
+	special = holder
+	special = ruler_child
+	tier = kingdom
+	priority = 42
+	culture_groups = { dragon_group }
+}
+princess_female_dragon_kingdom {
+	type = character
+	gender = female
+	special = ruler_child
+	tier = kingdom
+	priority = 42
+	top_liege = no
+	culture_groups = { dragon_group }
+}
```

Comment:
Don't get what is the point. Ruler children are called princes by default.

Reply:
> only in feudal

Add new government for dragons. Otherwise, your titles will backfire with mayors having children princes though they won't inherit anything.

Reply:
It's planned, it's like a placeholder.

Reply:
> It's planned, it's like a placeholder.

Don't leave placeholders on the master branch. As I said, it will backfire.

Reply:
> Am I commenting all of that ?

May comment, may delete, may move to another branch.

Reply:
Commented in edb864b18